### PR TITLE
fix(RemovalReasons): Hide "Add removal reason" button in modqueue

### LINKED ISSUE #169: "add removal reason" shows on comments where it shouldn't.
The button shouldn't show: 

- In queues. 
- When the user doesn't have removal reasons enabled for comments.

### PR DESCRIPTION
**Pull request by SpyTec**
fixes #169 

Also don't show "Add removal reason" for comments if the user setting is disabled

### CODE REVIEW

File: extension/data/modules/removalreasons.js [hunk 1]
```diff
@@ -161,19 +161,21 @@ function removalreasons () {
 
         // UI components
         // UI event handling
-        TB.listener.on('post', e => {
-            if (e.detail.data.isRemoved) {
-                const $target = $(e.target);
-                $target.append(`<span class="tb-bracket-button tb-add-removal-reason" data-id="${e.detail.data.id}" data-subreddit="${e.detail.data.subreddit.name}">Add removal reason</span>`);
-            }
-        });
+        if (TBUtils.pageDetails.pageType !== 'queueListing') {
```
Comment (creesch) [inline extension/data/modules/removalreasons.js]: This is at the wrong place, on redesign if someone starts with a queue page they never will get the notification. It needs to be switched with the line below.  And done separately for the comment listener.

Comment (SpyTec) [reply extension/data/modules/removalreasons.js]: Redesign, right. I'll move it to the listeners

**Comment by SpyTec** [comment]
Fixed feedback in latest commit

File: extension/data/modules/removalreasons.js (hunk 1 above)
Comment (creesch) [reply extension/data/modules/removalreasons.js]: 👍

**Pull request merged**